cloud/gcp/gcp.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+HOST=mar
+# IMAGE="ubuntu-1604-xenial-v20171107b"
+IMAGE_FAMILY='ubuntu-1604-lts'
+IMAGE_PROJECT='ubuntu-os-cloud' # image-project 
+TYPE="n1-standard-1"
+ZONE="us-west1-a"
+
+if [ "$#" -lt 1 ] 
+then
+  echo "Usage: gce.sh {add-disk | ssh | start | stop}"
+  exit 1 
+fi
+
+CMD="$1"
+
+if [ ${CMD}x = "add-disk"x ] 
```

Comment:
Suggesting using the more modern variant of conditions:
if [[ ${CMD} = "add-disk" ]]
(same below)